### PR TITLE
Add meeting notes 2020-01-07

### PR DESCRIPTION
**Pull request by chartgerink**
Crazy blue cat loves these notes ❤️

### CODE REVIEW

File: 20200107.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    - Update by Chris
+      - Bumped module spec to `v0.2.2`, now includes the avatar for profiles as well
+      - Text specification is still open; I think this will only start moving when we start producing content
+      - Interoperability spec has stalled; I'll be doing a refresh before our next meeting to make sure it's up-to-date
+    - Questions
+-   [`@p2pcommons/sdk-js`](https://github.com/p2pcommons/sdk-js)
+    - Update by Diego
+      - Migrating to latest corestores and hyperdrive
+      	- Lots of updates of dependencies, so we're trying to keep track
+        - Hyperswarm working, but using old version
```
Comment (dpaez) [inline 20200107.md]:
```suggestion
        - Swarm & replication working, but using old hyperdrive version (`^10.4.x`)
```

File: 20200107.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      - Having an issue with `mirror-folder`, need to do a deepdive on these
```
Comment (dpaez) [inline 20200107.md]:
```suggestion
      - Having some issues integrating `mirror-folder` with latest `hyperdrive`, need to do a deepdive on these
```

File: 20200107.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    - Update by Diego
+      - Migrating to latest corestores and hyperdrive
+      	- Lots of updates of dependencies, so we're trying to keep track
+        - Hyperswarm working, but using old version
+      - Having some issues integrating `mirror-folder` with latest `hyperdrive`, need to do a deepdive on these
+    - Questions
+-   [`@hypergraph-xyz/cli`](https://github.com/hypergraph-xyz/cli)
+    - Update by Julian
+      - Finished register action; released `v0.3.0`
+    - Questions
```
Comment (juliangruber) [inline 20200107.md]: shall we omit these if there were no questions, to improve the signal-to-noise ratio?

File: 20200107.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+-   [`@hypergraph-xyz/desktop`](https://github.com/hypergraph-xyz/desktop)
+    - Update by Julian
+      - Setup a workbench for isolated component development for Patch, [Storybook](storybook.js.org/)
```
Comment (juliangruber) [inline 20200107.md]:
```suggestion
      - Set up a workbench for isolated component development for Patch, [Storybook](storybook.js.org/)
```

tiny cute fix

File: 20200107.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    - Update by Julian
+      - No updates
+    - Questions
+-   [`@hypergraph-xyz/mobile`](https://github.com/hypergraph-xyz/mobile)
+    - PLACEHOLDER
+-   [`@hypergraph-xyz/vault`](https://github.com/hypergraph-xyz/vault)
+    - Update by Julian
+      - API part started; set up basic webserver and basic Stripe webhook endpoint
+      - Set up Dockerfile and deploy script; have deployed but not reachable
+      - When we decide to move away from AWS, we can move eg to Google Cloudrun, or self-hosted environment.
```
Comment (juliangruber) [inline 20200107.md]:
```suggestion
      - When we decide to move away from AWS, we can move eg to Google Cloud Run, or self-hosted environment.
```

File: 20200107.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+      - Serverless: We would be deploying application code, and not server operations. Problem for us is that it isn't portable. We could use serverless docker, which doesn't have this issue. We wouldn't need to manage the machines. Serverless abstracts the machine away
+      - If we want to manage our own machines we'd need an ops person.
+      - This would be a good moment to see how distributed filesystems (for DP). E.g., an s3 worker.
```
Comment (juliangruber) [inline 20200107.md]:
```suggestion
      - This would be a good moment to see how distributed filesystems work (for DP). E.g., an s3 worker.
```

**Pull request merged**